sw/device/lib/dif/dif_kmac_unittest.cc

```diff
@@ -103,7 +103,36 @@ class KmacTest : public testing::Test, public mock_mmio::MmioTest {
       0xa7, 0x48, 0x47, 0x93, 0x0a, 0x03, 0xab, 0xee, 0xa4,
       0x73, 0xe1, 0xf3, 0xdc, 0x30, 0xb8, 0x88, 0x15};
 
-  KmacTest() { EXPECT_DIF_OK(dif_kmac_init(dev().region(), &kmac_)); }
+  struct Config_register {
+    bool en;
```

Comment:
Suggestions for this struct:
- I'd give these fields default values to avoid nasty uninitialization surprises. That way you don't have to perform the big initialization block in the constructor (which aught to be a designated initializer, anyways).
- Prefer to avoid contractions line `en`.
- Many of these fields of uint8 type feel like they should be enums with clear names.
- The name should be in CamelCase.